_data/download_platforms.yml

```diff
@@ -14,30 +14,36 @@
 
 - name: "android.apk"
   title: "Android"
-  caption: "Universal APK (ARM64 + ARMv7 + x86_64 + x86)"
+  caption: "Universal APK"
   tags:
     - APK download
-    - ARM64/v7
-    - x86 (64 & 32 bit)
+    - ARM64
+    - ARMv7
+    - x86-64
+    - x86
 
 - name: "android.playstore"
   title: "Android"
-  caption: "Play Store Universal (ARM64 + ARMv7 + x86_64 + x86)"
+  caption: "Play Store Universal"
   tags:
     - Play Store
-    - ARM64/v7
-    - x86 (64 & 32 bit)
+    - ARM64
+    - ARMv7
+    - x86-64
+    - x86
 
 - name: "linux.32"
   title: "Linux"
   caption: "Standard (x86)"
   tags:
+    - x86
     - 32 bit
 
 - name: "linux.64"
   title: "Linux"
-  caption: "Standard (x86_64)"
+  caption: "Standard (x86-64)"
```

Comment:
We consistently use `x86_64` in Godot, AFAIK only macOS uses `x86-64` (which they now deprecated :upside_down_face: ), so I would undo that change and make the tag `x86_64`.

Reply:
Actually I'd suggest harmonizing all arch names and tags to the ones we use in Godot:
- `x86_64`
- `x86_32`
- `arm64`
- `arm32`